models/model.js

```diff
@@ -63,4 +63,41 @@ exports.updateVotebyArticleId = (voteChange, article_id, voteObjKey) => {
             const updatedArticle = result.rows[0];
             return updatedArticle;
         })
-}
\ No newline at end of file
+}
+
+exports.selectArticles = (topic) => {
+
+    const sort_by = 'created_at';
+    const order = 'DESC';
+
+    if (!['cats', 'mitch', 'paper'].includes(topic) && topic !== undefined) {
+        return Promise.reject({ status: 400, msg: "Topic doesn\'t exist!" });
+    }
+
+    const queryParams = [];
+
+    let sqlQuery = `SELECT a.author, a.title, a.article_id, a.body, a.topic, a.created_at, a.votes, count(c.body) ::INT AS comment_count
```

Comment:
for a query if you want to return all columns you can use "a.*" which cleans up your query

Reply:
or articles.*